src/etc/rust-gdb

```diff
@@ -21,4 +23,6 @@ RUST_GDB="${RUST_GDB:-gdb}"
 PYTHONPATH="$PYTHONPATH:$GDB_PYTHON_MODULE_DIRECTORY" exec ${RUST_GDB} \
   --directory="$GDB_PYTHON_MODULE_DIRECTORY" \
   -iex "add-auto-load-safe-path $GDB_PYTHON_MODULE_DIRECTORY" \
+  -iex "set substitute-path /rustc/$RUSTC_COMMIT_HASH $RUSTC_SYSROOT/lib/rustlib/src/rust" \
```

Comment:
This seems reasonable(ish), but I'm wondering whether this is a problem if the destination does not exist. Maybe it's clearer to see the hash path? Not sure.

Reply:
IME, gdb only uses the substitution for reading the source file, but still displays the original path in backtraces and such. Even the TUI layout shows the source window labeled with the original path.

Reply:
IIUC, there's no issue if a path doesn't exist, the substitutions just don't work.